### PR TITLE
"chown -R sentry /var/lib/sentry/files" takes a long time on some setups

### PR DESCRIPTION
**Pull request by stumbaumr**
As discussed in https://github.com/getsentry/onpremise/issues/239 with some setups this chown takes over 20 minutes.

Thanks for pulling and I hope this is the correct file...

### CODE REVIEW

**Comment by mattrobenolt** [comment]
We'd likely want this applied back to the `9.1` folder too, I assume.

**Comment by BYK** [comment]
@mattrobenolt isn't 9.1 on code-freeze ATM? Otherwise, I'd argue we'd need a 9.1.3 release?

**Comment by BYK** [comment]
@stumbaumr thanks a lot for the PR btw, this looks good to me! Once we resolve the question @mattrobenolt raised, I'll accept and merge.

**Comment by mattrobenolt** [comment]
We don’t version anything based on the packaging here. I’d just merge and update the 9.1.2 tag. Ideally, we’d have an independent packaging version, and we’d have like, 9.1.2-1 or something. But I don’t think it matters enough.

**Comment by BYK** [comment]
@mattrobenolt fair enough. @stumbaumr - would you mind applying the same change to the 9.1 folder so we can ship it to more people? :)

File: git/docker-entrypoint.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@ if [ "$1" = 'sentry' ]; then
 	set -- tini -- "$@"
 	if [ "$(id -u)" = '0' ]; then
 		mkdir -p "$SENTRY_FILESTORE_DIR"
-		chown -R sentry "$SENTRY_FILESTORE_DIR"
+		find "$SENTRY_FILESTORE_DIR" ! -user sentry -exec chown -R sentry {} \;
```
Comment (yosifkit) [inline git/docker-entrypoint.sh]: Wouldn't you want to drop the `-R` on the `chown`?

Comment (BYK) [reply git/docker-entrypoint.sh]: Good catch, I think so :)

Comment (BYK) [reply git/docker-entrypoint.sh]: @stumbaumr what do you think about this?

Comment (stumbaumr) [reply git/docker-entrypoint.sh]: Yes, that would make more sense.

**Pull request merged**